File: Cryptography and security libraries and algorithms/Python/Code complete/Block ciphers.py
import os
from cryptography.fernet import Fernet, Cipher, algorithms, modes
import pyaes
key1 = Fernet.generate_key()
key2 = os.urandom(32)
message = b'This is my secret message'
cipher1 = Cipher(algorithms.Blowfish(key1), mode=modes.GCM(b'12345678'), backend=default_backend()) 
cipher2 = pyaes.AESModeOfOperationGCM(key2) 

#Choose one cipher and encrypt the message
cipher_text = cipher1.encryptor().update(message) + cipher1.encryptor().finalize()
print("Blowfish Cipher Text: ", cipher_text) 
cipher_text = cipher2.encrypt(message) 
print("AES-256-GCM Cipher Text: ", cipher_text)

File: Cryptography and security libraries and algorithms/Javascript/Code complete/Block ciphers.js
const crypto = require('crypto');

const key1 = crypto.createHash('sha1').update('MY_SECRET_KEY').digest();

const cipher1 = crypto.createCipheriv('bf-gcm', key1, '12345678');

const key2 = crypto.randomBytes(32);

const cipher2 = crypto.createCipheriv('aes-256-gcm', key2, null);

const message = 'This is my secret message';

// Choose one cipher and encrypt the message
const encrypted = cipher1.update(message, 'utf8', 'base64');
console.log('Encrypted message: ', encrypted);
// Decrypt the message using the other cipher
const decrypted = cipher2.update(encrypted, 'base64', 'utf8');
console.log('Decrypted message: ', decrypted);
